lib/routes/nyaa/main.ts

```diff
@@ -32,7 +32,7 @@ async function handler(ctx) {
 
     const { query, username } = ctx.req.param();
 
-    const rootURL = ctx.routerPath.split('/')[1] === 'sukebei' ? 'https://sukebei.nyaa.si' : 'https://nyaa.si';
+    const rootURL = ctx.routerPath?.split('/')[1] === 'sukebei' ? 'https://sukebei.nyaa.si' : 'https://nyaa.si';
```

Comment:
This doesn't really fix the issue but make it always false. Since `ctx.routerPath` only exists in `koa`.
The equivalent of `ctx.routerPath` will be `getSubPath(ctx)`
https://github.com/DIYgod/RSSHub/blob/ef7ff4906237017ff317c51097386ff2803fcfcd/lib/utils/common-utils.ts#L30